openslides_backend/action/base.py

```diff
@@ -109,22 +113,58 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
         action classes.
 
         This can only be used if payload is a list.
+
+        This is called after initial validation, but before additional relation
+        validation.
         """
         return instance
 
     def validate_fields(self, instance: Dict[str, Any]) -> Dict[str, Any]:
         """
         Validates all model fields according to the model definition.
         """
-        for key, value in instance.items():
-            try:
-                field = self.model.get_field(key)
-            except ValueError:
-                # If the field doesn't exist, it's additional payload and we just ignore it
-                continue
-            instance[key] = field.validate(value)
+        for field_name, field in self.model.get_fields():
+            if field_name in instance:
+                instance[field_name] = field.validate(instance[field_name])
+        return instance
+
+    def validate_relation_fields(self, instance: Dict[str, Any]) -> Dict[str, Any]:
+        """
+        Validates all relation fields according to the model definition.
+        """
+        for field_name, field in self.model.get_relation_fields():
+            if field_name in instance and field.equal_fields:
+                if isinstance(field, BaseTemplateField):
+                    raise NotImplementedError()
+                fqids = self.get_field_value_as_fqid_list(field, instance[field_name])
+                for fqid in fqids:
+                    related_model = self.fetch_model(fqid, deepcopy(field.equal_fields))
```

Comment:
No, but the database `get` method does. It adds the `meta_position` field. I added the deepcopy here because it sufficed for this PR, but maybe we should rather move the deepcopy into the get method. What do you think?

Reply:
Sounds reasonable. Let's add deepcopy to the get method. The meta_position is handled by the datastore adapter.